burn-dataset/Cargo.toml

```diff
@@ -19,8 +19,8 @@ audio = [
 
 fake = ["dep:fake"]
 
-sqlite = ["__sqlite-shared", "dep:rusqlite"]
-sqlite-bundled = ["__sqlite-shared", "rusqlite/bundled"]
+sqlite = ["__sqlite-shared", "dep:rusqlite", "dep:gix-tempfile"]
+sqlite-bundled = ["__sqlite-shared", "rusqlite/bundled", "dep:gix-tempfile"]
 
 # internal
 __sqlite-shared = ["dep:r2d2", "dep:r2d2_sqlite", "dep:serde_rusqlite", "dep:image"]
```

Comment:
I would put the dependency in `__sqlite-shared`.

Reply:
Ah derp; of course!